scheduler/src/cook/mesos/util.clj

```diff
@@ -391,30 +453,37 @@
 ;; running and waiting jobs, so it's implemented that way to keep things simple.
 (defn get-active-jobs-by-user-and-state
   "Returns all jobs for a particular user in the specified state
-   and timeframe, without a custom executor.
-   Note that this query is not performant for completed jobs, use
-   get-completed-jobs-by-user instead."
-  [db user start end state name-filter-fn include-custom-executor?]
+   and timeframe, without a custom executor. This query works for state
```

Comment:
"without a custom executor" isn't really accurate since the caller gets to choose